transformation/plugins/org.jboss.tools.fuse.transformation.core/src/org/jboss/tools/fuse/transformation/core/model/xml/XmlModelGenerator.java

```diff
@@ -279,7 +287,15 @@ public JCodeModel generateFromInstance(File instanceFile,
         }
 
         // Step 2 - call generateFromSchema with generated schema
-        return generateFromSchema(schemaFile, packageName, targetPath);
+		final JCodeModel generatedSchema = generateFromSchema(schemaFile, packageName, targetPath);
+
+		if (initialValue_AccessExternalSchema != null) {
+			System.setProperty(JAVAX_XML_ACCESS_EXTERNAL_SCHEMA, initialValue_AccessExternalSchema);
+		} else {
+			System.clearProperty(JAVAX_XML_ACCESS_EXTERNAL_SCHEMA);
```

Comment:
Considering the discussion we had on FUSETOOLS-1463, do we really need to reset this? Your implementation is definitely a bit cleaner than my hack, so I'm good with it otherwise.